features/nanostack/coap-service/source/coap_security_handler.c

```diff
@@ -110,7 +122,11 @@ static int coap_security_handler_init(coap_security_t *sec)
 
     mbedtls_ssl_init(&sec->_ssl);
     mbedtls_ssl_config_init(&sec->_conf);
-    mbedtls_ctr_drbg_init(&sec->_ctr_drbg);
+#if defined(MBEDTLS_CTR_DRBG_C)
+    mbedtls_ctr_drbg_init(&sec->_drbg);
+#elif defined(MBEDTLS_HMAC_DRBG_C)
+    mbedtls_hmac_drbg_init(&sec->_drbg);
+#endif
```

Comment:
Similarly here #else ?

Reply:
No, we don't want to pollute code with dead code